### ISSUE TITLE
New maintainer?

### ISSUE THREAD

**Issue by jeremyheiler** [Raynes/bultitude#32]
@technomancy I see that you've committed directly to master on this project. Does this project need a new maintainer? 

Rest in peace, Raynes.

**Comment by technomancy** [comment]
Yeah, it would be great to have someone take over. I don't have the ability to add new contributors to this repo, but I can push changes to the readme file which could point users in the direction of a maintained fork.

**Comment by TimoFreiberg** [comment]
I'm making an attempt:
https://github.com/TimoFreiberg/bultitude/
https://clojars.org/timofreiberg/bultitude
